packages/manager/modules/pci/src/projects/project/data-processing/submit-job/price-details/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,24 @@
+{
+  "data_processing_price_details_estimation": "Estimation du prix",
+  "data_processing_price_details_description": "Le récapitulatif ci-dessous est une estimation du coût de votre job, et peut évoluer en fonction des resources utilisées.",
+  "data_processing_price_details_cores": "Cores",
+  "data_processing_price_details_memory": "Mémoire",
+  "data_processing_price_details_estimated_price": "Prix estimé",
+  "data_processing_price_details_driver": "Driver : ",
+  "data_processing_price_details_workers": "Executors : ",
+  "data_processing_price_details_memory_gib": "{{memory  | number:2}} GiB",
+  "data_processing_price_details_memory_overhead": "overhead",
+  "data_processing_price_details_price_per_core": "Prix par core : ",
+  "data_processing_price_details_price_per_gib": "Prix par GiB : ",
+  "data_processing_price_details_total": "Total : ",
+  "data_processing_price_details_unit_minute": "minute",
+  "data_processing_price_details_unit_hour": "heure",
+  "data_processing_price_details_unit_day": "jour",
+  "data_processing_price_details_unit_minute_plural": "minutes",
```

Comment:
**suggestion:** Can be maintained as constant.

Reply:
Hi @anooparveti, I think that those need to be translated, I am not certain we can say "minutes" in Espagnol or in Polish languages